### PR TITLE
Introduce dropFrame in VRBrowserState

### PR DESCRIPTION
**Pull request by mshin-wolvic**
This patch introduces `dropFrame` in VRBrowserState for chromium backend to discard the current frame and wait for the next tick of vsync. Also, this prevents from visiting StartFrame/EndFrame to keep the current scene when discarding frame.

### CODE REVIEW

File: app/src/main/cpp/ExternalVR.cpp [hunk 1]
```diff
@@ -602,6 +602,14 @@ ExternalVR::WaitFrameResult() {
       // VRB_LOG("RequestFrame BREAK %llu",  m.browser.layerState[0].layer_stereo_immersive.frameId);
       break;
     }
+
+#if CHROMIUM
+    if(m.browser.dropFame) {
```
Comment (zakharvoit) [inline app/src/main/cpp/ExternalVR.cpp:607]: nit: format code (missing space after 'if' before opening bracket).

**Review by zakharvoit — APPROVED** [review]
Approved with a small nit.

File: app/src/main/cpp/ExternalVR.cpp (hunk 1 above)
Comment (mshin-wolvic) [reply app/src/main/cpp/ExternalVR.cpp]: Done. Thanks

File: app/src/main/cpp/BrowserWorld.cpp [hunk 2]
```diff
@@ -1841,11 +1841,11 @@ BrowserWorld::TickImmersive() {
             DrawImmersive(aEye);
         };
       }
+      m.frameEndHandler = [=]() {
+        m.device->EndFrame(aDiscardFrame ? DeviceDelegate::FrameEndMode::DISCARD : DeviceDelegate::FrameEndMode::APPLY);
+        m.blitter->EndFrame();
+      };
```
Comment (svillar) [inline app/src/main/cpp/BrowserWorld.cpp:1847]: Hmm this is not correct, we should set the handler even when frameahead is not supported

**Review by svillar — CHANGES_REQUESTED** [review]

File: app/src/main/cpp/BrowserWorld.cpp (hunk 2 above)
Comment (mshin-wolvic) [reply app/src/main/cpp/BrowserWorld.cpp:1847]: Thanks for the comment, I will take a look more.

Comment (mshin-wolvic) [reply app/src/main/cpp/BrowserWorld.cpp]: Rollback the change in this file. Thanks!

**Pull request merged**